frontend/src/modules/common/components/BaseLogoNavDropdown/BaseLogoNavDropdown.jsx

```diff
@@ -44,21 +44,6 @@ const BaseLogoNavDropdown = ({ darkMode, showWorkflows = false, type = 'apps' })
           <span>Back to {isWorkflows ? 'workflows' : 'apps'}</span>
         </Link>
         <div className="divider"></div>
-        {isWorkflows || !showWorkflows ? (
-          <Link target="_blank" to={getPrivateRoute('dashboard')} className="dropdown-item tj-text tj-text-xsm">
-            <SolidIcon name={'apps'} width="20" fill="#C1C8CD" />
-            <span>{'Apps'}</span>
-          </Link>
-        ) : (
-          workflowsEnabled &&
```

Comment:
can we add back this workflows part ?

Reply:
FYI, Workflows is an enterprise feature. If it is enabled for the instance, it should be shown in the menu.

Reply:
If this is working fine for enterprise, you can ignore this change.